pep517/envbuild.py

```diff
@@ -90,9 +90,14 @@ def pip_install(self, reqs):
         if not reqs:
             return
         log.info('Calling pip to install %s', reqs)
-        check_call([
+        cmd = [
             sys.executable, '-m', 'pip', 'install', '--ignore-installed',
-            '--prefix', self.path] + list(reqs))
+            '--prefix', self.path] + list(reqs)
+        check_call(
```

Comment:
will this print out the output if the call fails? 🤔

Reply:
It will not. Previously, the output would be emitted unconditionally. Now the output will be redirected to the log, where it can be controlled based on logging settings. I think this library probably needs to step back and specify what callers should expect about output across various operations.

Reply:
I'm hoping not to have to solve that problem here but only provide a reasonable behavior and avoid emitting the output for API callers.

Reply:
This issue is being addressed more generally in #13.